gallery.html

```diff
@@ -31,7 +31,7 @@
         <header id="top">
             <nav>
                 <input type="checkbox" id="menu-toggle" name="sandwich">
-                <label class="sandwich-icon" for="checkbox"></label>
+                <label class="sandwich-icon" for="menu-toggle"></label>
```

Comment:
The `for` attribute must refer to the `id` of the control associated with the label